### PR TITLE
Handle the destruction of Projector Window (fix a crash)

### PR DESCRIPTION
**Pull request by RodrigoHolztrattner**
Asana task: https://app.asana.com/0/734357654754972/1107505184397281/f

The window HWND is attached to the projector HTML DOM, we cannot control its destruction so we must allow the `DestroyWindow()` method to fail with error 1400, invalid window reference, if it (HTML DOM) was able to destroy it first.

### CODE REVIEW

**Comment by summeroff** [comment]
In case someone else try to close our window we can catch that be looking for messages WM_CLOSE and WM_DESTROY in OBS::Display::DisplayWndProc . And delete Display associated with destroyed HWND.  

It is more suggestion for an improvement as fix in this PR is ok and it small and fixes crash without increasing chances of side effects.

**Comment by RodrigoHolztrattner** [comment]
Good suggestion! I will add it as a different task on Sentry since the reason for this PR is to fix the crash but would be nice to update the code to handle the way you said.

Update: Asana task: https://app.asana.com/0/734357654754972/1108635072226916/f

File: obs-studio-server/source/nodeobs_display.cpp [hunk 1]
```diff
@@ -205,8 +204,19 @@ void OBS::Display::SystemWorker()
...
+                // We check for error 1400 because if this display is a projector, it is attached to a HTML DOM, so
+                // we cannot directly control its destruction since the HTML will probably do this concurrently, 
+                // the DestroyWindow is allows to fail on this case, a better solution here woul be checking if this
+                // display is really a projector and do not attempt to destroy it (let the HTML do it for us).
+				if (error != 1400) {
+					answer->success = false;
+					HandleWin32ErrorMessage(error);
+                } else {
+					answer->success = true;
+                }
```
Comment (EddyGharbi) [inline obs-studio-server/source/nodeobs_display.cpp]: Please fix indentation here

Comment (RodrigoHolztrattner) [reply obs-studio-server/source/nodeobs_display.cpp]: Done

**Pull request merged**